plugins/example-todo-list-common/package.json

```diff
@@ -12,7 +12,8 @@
   },
   "publishConfig": {
     "access": "public",
-    "main": "dist/index.esm.js",
+    "main": "dist/index.cjs.js",
+    "module": "dist/index.esm.js",
```

Comment:
This package it private and doesn't get published, it's really here to help with the [Permissions tutorial](https://backstage.io/docs/permissions/plugin-authors/01-setup). This change make sense to me but I am curious why would you be building a Docker image of this entire repo? It's really not intended to be used that way.

Reply:
Does that matter when its referenced in documentation, https://backstage.io/docs/getting-started/configure-app-with-plugins, which would probably give people the idea they can use it as a boilerplate? A problem you can see here @ https://github.com/backstage/community-plugins/blob/0e05bd93261bbe897086d4a63beffa568c564d9f/workspaces/gocd/plugins/gocd/package.json#L14

Considering the above imho it make sense to fix? If we want people to base their work on our things, we need to make sure they dont end up spending days running in circles on a ESM issue that makes no sense because they decided to use the multi-stage Dockerfile from the docs in a backend that was generated by the CLI :P

With regards to the Dockerfile, im building it in an app repo, not the full mon repo, with that said you can build container images off the stuff in the main repo, I have done so before https://github.com/dfds/backstage_originaldfdsfork/tree/main/dfds-deploy, but that is besides the point since this PR is not really related to that. :)

And thank you for your feedback <3

Reply:
I'm totally fine with this change, sorry if that wasn't clear. The part about building Docker images of this repo did concern me as it's not recommended and we see a lot of people doing that which leads to problems for them. But based on your comment you are using `create-app` as your base so that's fine and I'm much less concerned 👍